x-pack/plugins/enterprise_search/server/routes/app_search/crawler.ts

```diff
@@ -66,6 +66,21 @@ export function registerCrawlerRoutes({
     })
   );
 
+  router.get(
+    {
+      path: '/api/app_search/engines/{name}/crawler/domains/{id}',
```

Comment:
The name and id params here read a bit weird to me because they could appear to be related but one if the engine name and the other is the domain ID. I see this is consistent with the other routes but perhaps something to look into and clear up later.

Reply:
Yeah this follows existing but not great patterns, in the future I would like us to be more specific with these names.